WoWPro_Leveling/Wrath/Neutral/WOTLK_ZulDrak.lua

```diff
@@ -68,62 +68,62 @@ C Dressing Down|QID|12648|N|Return to the Reliquary of Agony and use the choker.
 T Close Call|QID|12638|M|14.1,73.8|N|To Stefan Vadu.|
 A Silver Lining|QID|12643|PRE|12638|N|From Stefan Vadu.|
 C Silver Lining|QID|12643|N|Around the Reliquary of Agony . Pick up the Drool from the ground. Kill the Abominations.|M|21.24,75.01|
-T Silver Lining|QID|12643|N|Back to Ebon Watch.|M|14.12,73.79|
+T Silver Lining|QID|12643|N|Back to Stefan Vadu.|M|14.12,73.79|
 A Suit Up!|QID|12649|O|PRE|12643|N|From Stefan Vadu.|
 C Suit Up!|QID|12649|O|N|Return to the Reliquary of Agony and use the choker. You cannot use while shapeshifted. Buy the plasma from Gristlegut.|U|38699|M|19.91,75.44|
 A Feedin' Da Goolz|QID|12652|M|19.9,75.5|N|From Gristlegut.|FACTION|Horde|
-C Feedin' Da Goolz|QID|12652|N|Use the bowl near the ghouls. Be careful of the Blightguards, they see through your disguise. You should hotkey the chocker because your diguise will fail at some point.|U|38701|
+C Feedin' Da Goolz|QID|12652|N|Use the bowl near the ghouls. Be careful of the Blightguards, they see through your disguise. You should hotkey the chocker because your diguise will fail at some point.|U|38701|M|21.24,75.01|
```

Comment:
Add a `U` tag for the chocker and eliminate the hotkey comment.

Reply:
Can you have 2 U?

Reply:
Not on one step, but make the chocker a different step.